FreeRTOS/FreeRTOS
# Add CBMC proof for prvProcessEthernetPacket

Description
-----------
This PR adds the CBMC proof for the prvProcessEthernetPacket function present in the FreeRTOS_IP.c.

Test Steps
-----------
- Download the repo with `--recurse-submodules`. Switch to the correct branch.
- Switch to `FreeRTOS-Plus/Test/CBMC/proofs` directory and run `python3 prepare.py`.
- Move to `prvProcessEthernetPacket` directory. Run `make cbmc`.
- Open the cbmc.txt file generated by the above command and verify that there were no failures.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
